OrbitCore/SymbolHelper.cpp

```diff
@@ -131,6 +131,13 @@ bool SymbolHelper::LoadSymbolsIncludedInBinary(
 bool SymbolHelper::LoadSymbolsCollector(std::shared_ptr<Module> module) const {
   if (LoadSymbolsIncludedInBinary(module)) return true;
 
+  if (module->m_DebugSignature.empty()) {
+    LOG("symbol loading from a separate file is not supported for module "
+        "\"%s\", because it does not contain a build id",
```

Comment:
maybe suggest some action of how to fix it to developer? recompile with newer toolchain/lld?

Reply:
I can write a little more, but I don't know if a suggestion like that actually helps anyone. I think the build id is default in gcc and clang, so if it is missing, that means somewhere it has been turned off.